kombu/transport/azurestoragequeues.py

```diff
@@ -59,11 +60,11 @@
 class Channel(virtual.Channel):
     """Azure Storage Queues channel."""
 
-    domain_format = 'kombu%(vhost)s'
-    _queue_service = None
-    _queue_name_cache = {}
-    no_ack = True
-    _noack_queues = set()
+    domain_format: str = 'kombu%(vhost)s'
+    _queue_service: QueueServiceClient | None = None
```

Comment:
Shouldn't _queue_service be [Optional](https://docs.python.org/3/library/typing.html#typing.Optional)?

Reply:
I agree, but the pre-commit.ci automatically corrected it and switched it from `Optional[QueueServiceClient]` to this. I just assumed this was an alternative way to declare the same thing, but outside of this project I would always use the `Optional[QueueServiceClient]` notation.

What should I do?

Reply:
I made the updates, and I assume pre-commit.ci will add a commit correcting it, but we'll see.

Reply:
Yeah, it corrected it so I just re-pushed the original content that you commented on. Should I try to update the pre-commit.ci configuration or merge as is?

Reply:
If you can update the pre-commit config that would be great but I wouldn't block merging this PR without that config change.